src/main/java/core/basesyntax/storage/Storage.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.storage;
+
+import core.basesyntax.model.Fruit;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Storage {
+    public static final List<Fruit> fruits = new ArrayList<>();
```

Comment:
The access to the 'fruits' list should be encapsulated. It's a bad practice to have a public static field accessible from anywhere in the code. Consider providing methods to interact with the storage instead of exposing the list directly.